bindings/python/src/PyXRootDModule.cc

```diff
@@ -80,7 +80,6 @@ namespace PyXRootD
 #endif
   {
     // Ensure GIL state is initialized
-    Py_Initialize();
```

Comment:
To the best of my knowledge this is only needed when embedding a Python interpreter inside your application: https://docs.python.org/3/c-api/init.html